app/controllers/labware_controller.rb

```diff
@@ -31,7 +31,7 @@ def index # rubocop:todo Metrics/AbcSize, Metrics/MethodLength
   def show
     @source_plates = @asset.source_plates
     respond_to do |format|
-      format.html
+      format.html { @aliquots = @asset.aliquots.include_summary.paginate(page: params[:page], per_page: 384) }
```

Comment:
this change is going to be present in all labware; can we move this to a helper referenced in the view?